ui/imports/utils/Constants.qml

```diff
@@ -1113,7 +1113,19 @@ QtObject {
         MembershipRequests,
         RejectedMembers,
         BannedMembers
-    }   
+    }
+
+    enum CommunityMembershipRequestState {
+        Pending,
```

Comment:
This is imho missing (and should start with) 
```qml
None = 0,
```
so that it mirrors the NIM enum

Reply:
Good point, thanks!